edge-apps/weather/index.html

```diff
@@ -15,7 +15,7 @@
                 tag_manager_id: tagManagerId,
             } = screenly.settings;
 
-            if (disableAnalytics !== 'true') {
+            if (disableAnalytics != undefined && disableAnalytics !== 'true') {
```

Comment:
It's not obvious that undefined == true. Why do we need this? Probably makes sense to use enableAnalytics instead. So Analytics will be disabled by default.

```suggestion
            if (enableAnalytics === 'true') {
```

Reply:
Just use default_value in screenly.yml to set it to true

Reply:
Yeah, you're right. It makes the setting more easy to use. Thanks.